src/Ledger/Certs/Properties.agda

```diff
@@ -246,10 +254,12 @@ module _  {Γ : CertEnv}
     CERTBASE-pov :  {s s' : CertState} → Γ ⊢ s ⇀⦇ _ ,CERTBASE⦈ s'
                     → getCoin s ≡ getCoin s' + getCoin (CertEnv.wdrls Γ)
 
-    CERTBASE-pov  {s  = ⟦ ⟦ _ , _ , rewards  ⟧ᵈ , stᵖ , ⟦ dreps , ccHotKeys ⟧ᵛ ⟧ᶜˢ}
-                  {s' = ⟦ ⟦ _ , _ , rewards' ⟧ᵈ , stᵖ , stᵍ ⟧ᶜˢ}
+    CERTBASE-pov  {s  = cs}
+                  {s' = cs'}
                   (CERT-base {pp}{vs}{e}{dreps}{wdrls} (_ , wdrlsCC⊆rwds)) =
       let
+        open DState (dState cs )
+        open DState (dState cs') renaming (rewards to rewards')
         module ≡ᵉ       = IsEquivalence (≡ᵉ-isEquivalence {Credential × Coin})
         wdrlsCC         = mapˢ (map₁ RwdAddr.stake) (wdrls ˢ)
         zeroMap         = constMap (mapˢ RwdAddr.stake (dom wdrls)) 0
```

Comment:
These changes to `Ledger.Certs.Properties` improve overall clarity and readability.  Nice.